ams/server/src/main/java/com/netease/arctic/server/dashboard/MixedAndIcebergTableDescriptor.java

```diff
@@ -517,13 +505,18 @@ private List<PartitionFileBaseInfo> collectFileInfo(
                 dataFileType,
                 commitTime,
                 partitionPath,
-                filePath,
+                contentFile.specId(),
+                contentFile.path().toString(),
                 fileSize));
       }
-    } catch (IOException exception) {
-      LOG.error("close manifest file error", exception);
+      return result;
+    } finally {
+      try {
+        entries.close();
+      } catch (IOException e) {
+        LOG.warn(e.getMessage(), e);
```

Comment:
The exception error message will be printed in the trace, so it is not recommended to display the error message again.
```suggestion
        LOG.warn("Failed to close the manifest reader.", e);
```